dnf/cli/commands/module.py

```diff
@@ -115,6 +123,14 @@ def run_on_module(self):
                             libdnf.module.ModulePackageContainer.ModuleErrorType_ERROR_IN_DEFAULTS:
                         raise e
                 logger.error(str(e))
+            switchedModules = dict(self.base._moduleContainer.getSwitchedStreams())
+            if switchedModules:
+                report_module_switch(switchedModules)
+                msg = _("The module stream switching is not allowed operation.\n"
+                        "It is reccomend to remove all installed content from the module and then "
+                        "reset the module using 'dnf module reset <module_name>' command.\n"
+                        "The reseting of the module allows to enable a different stream.")
```

Comment:
Just a couple of minor typos:

    s/reccomend/recommended/g
    s/reseting/resetting/g

I would also recommend changing some of the supporting statements:

                msg = _("Switching between module streams is not allowed in a single operation.\n"
                        "Removing all installed content from the installed module is recommended, then "
                        "reset the module using the 'dnf module reset <module_name>' command.\n"
                        "The resetting of the module allows a different stream to be enabled.")

Reply:
@kyle-walker I am open for any further suggestions